docs/logcli.md

```diff
@@ -20,6 +20,12 @@ Common labels: {job="cortex-ops/consul", namespace="cortex-ops"}
 2018-06-25T12:52:09Z {instance="consul-8576459955-pl75w"} 2018/06/25 12:52:09 [INFO] raft: Snapshot to 475409 complete
 2018-06-25T12:52:09Z {instance="consul-8576459955-pl75w"} 2018/06/25 12:52:09 [INFO] raft: Compacting logs from 456973 to 465169
 ```
+If you want to use config stored in file, you need to specify whole config file path with `--config` option, please take `cmd/logcli/logcli-config.yaml` as example.
```

Comment:
This sentence needs cleaning up: 
"You may use `--config=path/to/file` to load configuration options from a file. For an example file see `cmd/logcli/logcli-config.yaml`.